sqlmesh/core/engine_adapter/base.py

```diff
@@ -1319,7 +1319,7 @@ def _scd_type_2(
             update_valid_from_start = self._to_utc_timestamp(
                 "1970-01-01 00:00:00+00:00", time_data_type
             )
-        insert_valid_from_start = execution_ts if check_columns else updated_at_name
+        insert_valid_from_start = execution_ts if check_columns else exp.to_column(updated_at_name)  # type: ignore
```

Comment:
should this use column and not to_column? what are the expected inputs of updated_at_name?

Reply:
It is just a string representing the column name so yeah `exp.column` seems like a better fit